homework/index.js

```diff
@@ -1,53 +1,143 @@
 'use strict';
 
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status >= 200 && xhr.status <= 299) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
-      }
-    };
-    xhr.onerror = () => cb(new Error('Network request failed'));
-    xhr.send();
+  function fetchJSON(url) {
+    return new Promise((resolve, reject) => {
+      const xhr = new XMLHttpRequest();
+      xhr.open('GET', url);
+      xhr.responseType = 'json';
+      xhr.onload = () => {
+        if (xhr.status >= 200 && xhr.status <= 299) {
+          resolve(xhr.response);
+        } else {
+          reject(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+        }
+      };
+      xhr.onerror = () => reject(new Error('Network request failed'));
+      xhr.send();
+    });
   }
 
   function createAndAppend(name, parent, options = {}) {
     const elem = document.createElement(name);
     parent.appendChild(elem);
-    Object.entries(options).forEach(([key, value]) => {
+    Object.entries(options).forEach(([key, content]) => {
       if (key === 'text') {
-        elem.textContent = value;
+        elem.textContent = content;
       } else {
-        elem.setAttribute(key, value);
+        elem.setAttribute(key, content);
       }
     });
     return elem;
   }
 
+  function changeDateFormat(date) {
+    const timeFormat = new Date(date);
+    return timeFormat.toLocaleString();
+  }
+
+  function createTable(table, header, content) {
```

Comment:
This function does not create a table: it creates a table **row**. So a more precise name would be `createTableRow`.